packages/paste-website/src/pages/components/visual-picker/index.mdx

```diff
@@ -0,0 +1,418 @@
+
+Always use the `labelText` prop on VisualPickerRadio and VisualPickerCheckbox to provide a descriptive string for each option.
+
+When focused on a Visual Picker Radio control, use arrow keys to navigate to the other radios. When focused on a Visual Picker Checkbox control, use the tab key to navigate to the other checkboxes.
+
+## Examples
+
+### Visual Picker Radio
+
+Use a Visual Picker Radio when a user must select a single option.
+
+#### Vertical
+
+Vertical is the default orientation for Visual Picker.
+
+<LivePreview
+  scope={{
+    VisualPickerRadioGroup,
+    VisualPickerRadio,
+    useUID,
+    Text,
+    Box,
+    MediaObject,
+    MediaFigure,
+    MediaBody,
+    Avatar,
+    AvatarGroup,
+    StatusBadge,
+    CodeIcon,
+    CommunityIcon,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerRadioVertical}
+</LivePreview>
+
+#### Horizontal
+
+Use the horizontal Visual Picker when more complex visuals will help users compare information. Avoid using horizontal if there are more than 3-4 options.
+
+<LivePreview
+  scope={{
+    VisualPickerRadioGroup,
+    VisualPickerRadio,
+    useUID,
+    Text,
+    Box,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerRadioHorizontal}
+</LivePreview>
+
+### Visual Picker Checkbox
+
+Use a Visual Picker Checkbox when a user can select multiple options.
+
+#### Vertical
+
+Vertical is the default orientation for Visual Picker.
+
+<LivePreview
+  scope={{
+    VisualPickerCheckboxGroup,
+    VisualPickerCheckbox,
+    useUID,
+    Text,
+    Box,
+    MediaObject,
+    MediaFigure,
+    MediaBody,
+    Avatar,
+    AvatarGroup,
+    UserIcon,
+    SMSIcon,
+    CodeIcon,
+    VoiceCapableIcon,
+    ProductVideoIcon,
+    EmailIcon,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerCheckboxVertical}
+</LivePreview>
+
+#### Horizontal
+
+Use the horizontal Visual Picker when more complex visuals will help users compare information. Avoid using horizontal if there are more than 3-4 options.
+
+<LivePreview
+  scope={{
+    VisualPickerCheckboxGroup,
+    VisualPickerCheckbox,
+    useUID,
+    Text,
+    Box,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerCheckboxHorizontal}
+</LivePreview>
+
+### Internationalization
+
+Internationalizing Visual Picker is the same for both Radio and Checkbox Visual Pickers. To internationalize a Visual Picker, simply pass translated text as children. The only exception to this is the required dot in the legend of a required Visual Picker. To change the required dot's text, use the i18nRequiredLabel prop. Hover over the required dot to see the translated string.
+
+<LivePreview
+  scope={{
+    VisualPickerCheckboxGroup,
+    VisualPickerCheckbox,
+    useUID,
+    Text,
+    Anchor,
+  }}
+  noInline
+  showOverflow
+>
+  {VisualPickerCheckboxI18n}
+</LivePreview>
+
+## States
+
+### Checked
+
+Use a Visual Picker with a pre-checked value only if:
+
+- The default value would not cause harm if accidentally used
```

Comment:
Can we add periods to each of these (they're each a full sentence, so calls for a period)